src/vorta/application.py

```diff
@@ -52,49 +52,50 @@ def __init__(self, args_raw, single_app=False):
             elif args.profile:
                 self.sendMessage(f"create {args.profile}")
                 logger.info('Creating backup using existing Vorta instance.')
-                sys.exit()
         elif args.profile:
             sys.exit('Vorta must already be running for --create to work')
 
-        init_translations(self)
+        self.message_received_event.connect(self.message_received_event_response)
+        if not args.profile:
```

Comment:
`if True` is generally quicker to read than `if not XX`. And since we have a large code block under this condition now, it should get at least a comment on *when* this path is chosen. Still, keeping most of the code on the default path and dealing with exceptions some other way is preferable.